services/diversity/api_doc/richness.js

```diff
@@ -77,3 +77,18 @@
  *   }
  * ]
  */
+
+/**
+ * @apiDefine NOSLayerExample
+ * @apiSuccessExample {binary} Success-Example:
+ * image/png
+ */
+
+/**
+ * @apiDefine NSThresholdsExample
```

Comment:
This name is the same as the defined in line 38, I suppose the correct name is `NOSLayerThresholdsExample` ?

Reply:
Yes, fixed!